source/neuropod/bindings/java/src/main/native/utils.cc

```diff
@@ -0,0 +1,27 @@
+#include "utils.h"
+
+#include "neuropod/neuropod.hh"
+
+#include <iostream>
+#include <sstream>
+#include <stdexcept>
+#include <string>
+#include <vector>
```

Comment:
It includes a lot of unused headers. In C++ best practice is to include only explicitly used headers, I see std:string and jni.h only